pkg/sources/github/github.go

```diff
@@ -744,6 +757,25 @@ func (s *Source) getReposByUser(ctx context.Context, user string) ([]string, err
 	return repos, nil
 }
 
+func (s *Source) includeRepo(r string) bool {
+	if len(s.includeRepos) == 0 {
+		return true
+	}
+
+	for _, include := range s.includeRepos {
+		g, err := glob.Compile(include)
+		if err != nil {
+			s.log.WithError(err).Errorf("invalid glob %s", include)
```

Comment:
I think it would be better to log it at a different level for enterprise to cut down on the non-dev actionable errors.

Reply:
that works with me. I think i copied this logic from below on L783 which is probably why its logged at this level. Might be worth at some point looking through all the logs in here.

Reply:
Oh yeah. I think we use error quite a bit. Just had this fresh on my mind after the retro.